_posts/2018-07-02-blockchain-sharing-economy-autonomes-fahren.md

```diff
@@ -0,0 +1,57 @@
+(können), leitet uns der Lean Startup Ansatz an, wie wir durch Experimente Einblicke in noch nicht artikulierte Bedürfnisse 
+können.
+
+Als Coach in digitalen Innovationsprojekten ist es immer wieder meine Aufgabe, den Irrtum hinter dem Satz – wir kennen unsere 
+Kunden seit 30 oder mehr Jahren zu erklären. Ja, der Vertrieb hört genau auf die Wünsche Ihrer Kunden, doch beziehen sich 
+diese Wünsche in der Regel auf das bestehende Angebot: etwas mehr, etwas schneller und vor allem günstiger. Das verrät mir 
+wenig über digitale Nutzungsszenarien auf Basis bisher unbekannter Technologien.
+
+Der Lean Startup Ansatz macht sich bezahlt, wenn es um völlig neuartige, hoch risikobehaftete Produkt- oder 
+Serviceentwicklungen geht. Nahezu jede etwas längere Unternehmensgeschichte kennt gescheiterte Produktentwicklungen. 
+Oftmals sind große Anstrengungen unternommen worden, fähige Technologen und Designer mit viel Engagement und Zeit am Werk 
+gewesen, doch der Produktstart geriet zu einer Enttäuschung. Marketing und Vertrieb vermochten es nicht, das neue Produkt 
+dem Kundenkreis schmackhaft zu machen. Nicht weil sie die falschen oder zu wenig Mittel eingesetzt hätten, sondern weil 
+der erhoffte Shooting-Start ins Leere lief, schlicht kein Bedarf vorhanden war.
+
+Aus eigener, bitterer Erfahrung und Beobachtung vieler Startups in ihrer Umgebung entwickelten Steve Blank und Eric Ries Lean 
+Startup als Methode zur Entwicklung neuer Geschäftsmodelle. Neben einem innovativen Produkt gehören zu einem erfolgreichen 
+Geschäftsmodell eine ausreichend große Nachfrage. Der Lean Startup Ansatz richtet sich in der Hauptsache darauf, diese 
+potentielle Nachfrage genau genug zu untersuchen, sie verlässlich zu validieren, bevor hohe Investitionen in die 
+Produktentwicklung und Vermarktung getätigt werden. Mit Werkzeugen aus der Naturwissenschaft dem Hypothesentest und 
+Experiment werden die Bestandteile eines zukünftigen Geschäftsmodells experimentell untersucht, um Fehlinvestitionen zu 
+vermeiden und die wertvolle Zeit der Entwicklerteams auf die aussichtsreichen und gut verstandenen Chancen zu setzen.
+
+
+##Lean Startup Experiment
+
+In der Startup-Welt gehört der Lean Ansatz inzwischen zum Selbstverständnis. In etablierten Unternehmen findet er gerade 
+erst Einzug. Hier bedarf es noch einiger Überzeugungsarbeit, damit Innovationsteams vor der eigentlichen Entwicklungsarbeit 
+mit noch unausgereiften Ideen Experimente am Markt durchführen dürfen. Ein Business Model Canvas im Workshop wird akzeptiert, 
+doch raus aus dem Gebäude? Viele, viele Hürden und Prozessvorschriften sind hier zu überwinden: angefangen von Grundsätzen wie 
+One-Face to the Customer, Corporate Identity, Qualitätssicherung…
+
+Beim ![XCamp](https://xcamp.co/netvis/index) stärken wir allen, die den Lean Weg gehen den Rücken, tauschen uns über Argumente und Praxiserfahrungen aus. Wie 
```

Comment:
The exclamation mark means, that this is a picture. We want a normal link here instead.

Reply:
I again forgot that, sorry!